src/lib.rs

```diff
@@ -652,19 +694,26 @@ impl<U> File<U> {
 
 impl TextFile {
     fn make_change(&mut self, changes: &[&Change]) -> Result<(), Error> {
+        trace!("TextFile::make_change");
         for c in changes {
+            trace!("TextFile::make_change: {:?}", c);
             let new_text = match **c {
                 Change::ReplaceText {
                     ref span,
-                    ref len,
                     ref text,
                 } => {
+                    let byte_in_str = match span {
+                        VfsSpan::UnicodeScalarValue(..) => byte_in_str,
+                        VfsSpan::Utf16CodeUnit(..) => byte_in_str_utf16,
+                    };
```

Comment:
I would make `byte_in_str` a method on VfsSpan and match there, rather than using a function in a variable.

Reply:
Done: https://github.com/nrc/rls-vfs/pull/24/commits/b2ad8ad34fe2e4220082260cf1c4765a19c82ca8#diff-b4aea3e418ccdb71239b96952d9cddb6R63